app/uk/gov/hmrc/nisp/controllers/ExclusionController.scala

```diff
@@ -66,15 +66,15 @@ trait ExclusionController extends NispFrontendController with AuthorisedForNisp
   }
 
   def showNI: Action[AnyContent] = AuthorisedByAny.async { implicit user => implicit request =>
-     nispConnector.connectToGetNIResponse(user.nino).map {
-        case NIResponse(_, _, Some(niExclusions: ExclusionsModel)) =>
-          if(niExclusions.exclusions.contains(Exclusion.Dead)) {
-            Ok(excluded_dead(niExclusions.exclusions, None))
+     nationalInsuranceService.getSummary(user.nino).map {
+        case Left(exclusion) =>
+          if(exclusion == Exclusion.Dead) {
+            Ok(excluded_dead(List(Exclusion.Dead), None))
           }
-          else if(niExclusions.exclusions.contains(Exclusion.ManualCorrespondenceIndicator)) {
-            Ok(excluded_mci(niExclusions.exclusions, None))
+          else if(exclusion == Exclusion.ManualCorrespondenceIndicator) {
+            Ok(excluded_mci(List(Exclusion.ManualCorrespondenceIndicator), None))
           } else {
-            Ok(excluded_ni(niExclusions))
+            Ok(excluded_ni(exclusion))
```

Comment:
There is only one exclusion now rather than a list of NI specific information, I named it after the type which is just the enum